schema/ethernet.yml

```diff
@@ -367,4 +367,84 @@ properties:
       lldp-notification:
         type: boolean
         description: Enables the transmission of SNMP trap notifications about LLDP changes.
-        default: false
\ No newline at end of file
+        default: false
+  acl:
+    description: A collection of access control entries that define the rules for filtering traffic through a network port.
+    type: array
+    items:
+      type: object
+      properties:
+        acl-inf-policy-preference:
+          description: Determines the priority of multiple ACL policies when more than one is applied to an interface, if any.
+          type: integer
+          minimum: 1
+          maximum: 64
+          default: 1
+        acl-inf-policy-ingress:
+          description: Specifies the ACL policy that is applied to incoming traffic on an interface.
+          type: string
+          maxLength: 32
+          minLength: 1
+          examples:
+            - blacklisted-macs
+        acl-inf-counters-ingress:
+          description: Tracks the number and type of packets that match the ingress ACL rules on an interface.
+          type: boolean
+          default: false
+        acl-inf-policy-egress:
+          description: Specifies the ACL policy that is applied to outgoing traffic from an interface.
+          type: string
+          maxLength: 32
+          minLength: 1
+          examples:
+            - blacklisted-macs
+        acl-inf-counters-egress:
+          description: Tracks the number and type of packets that match the egress ACL rules on an interface.
+          type: boolean
+          default: false
+  voice-vlan-intf-config:
+    description: Configure the Voice VLAN feature at the interface level, allowing for VoIP traffic to be prioritized on this specific port.
+    type: object
+    properties:
+      voice-vlan-intf-mode:
+        description: Specify the mode of placing this port on the voice VLAN.
+        type: string
+        default: "auto"
+        enum:
+          - none
+          - manual
+          - auto
+      voice-vlan-intf-priority:
+        description: Define the Class of Service (CoS) priority for VoIP traffic passing through this port, ensuring higher priority over other traffic types.
+        type: integer
+        default: 6
+        minimum: 0
+        maximum: 6
+      voice-vlan-intf-detect-voice:
+        description: Select the detection method for identifying VoIP traffic on this port, such as OUI-based detection or traffic pattern recognition.
+        type: string
+        default: "oui"
+        enum:
+          - oui
+          - lldp
+      voice-vlan-intf-security:
+        description: Enable or configure security filtering for VoIP traffic on the interface to protect against unauthorized devices.
+        type: boolean
+        default: false
+dhcp-snoop-port:
```

Comment:
Indentation here is wrong. dhcp-snoop-port will not be taken as a property.
The joys of yaml